core/string/ustring.h

```diff
@@ -431,6 +431,8 @@ class String {
 	String replace_first(const char *p_key, const char *p_with) const;
 	String replace(const String &p_key, const String &p_with) const;
 	String replace(const char *p_key, const char *p_with) const;
+	String replace_char(char32_t p_key, char32_t p_with) const;
+	String replace_chars(const Vector<char32_t> &p_keys, char32_t p_with) const;
```

Comment:
How about taking in a `String` instead of a `Vector<char32_t>`? I'm not sure which is better; technically I would advocate taking in a `Span<const char32_t>`, but this is a) not possible yet and b) not exposeable to scripting.

Reply:
Good question, both would be useful, and that would allow us to expose it to scripting, will investigate

Reply:
Might even be worth making a `replace_chars(const char *p_key, char32_t p_with)` where we treat it the same way as we do in `find` etc., that'd make the literal cases easier

Will test and see, it'd make for nicer and faster use in specific cases if we can just use a `replace_chars("ABC", 't')` where we only care about Latin1, which is most if not all uses

Thank you for this idea, will add it as an optional addition to this later today